src/ts/background/services/keyring-vault.ts

```diff
@@ -89,11 +90,19 @@ class KeyringVault {
 
   getAccounts (): Array<KeyringPair$Json> {
```

Comment:
we probably should rename this to getKeyringPairs and other one will be getAccounts

Reply:
A lot of other functions already use account such as createAccount, updateAccountName, etc. Probably we can change Accounts type to SimpleAccounts

Reply:
yeah.

Reply:
updated